py/src/nupic/examples/tm/tm_high_order.py

```diff
@@ -0,0 +1,307 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.
+#               2019, David McDougall
+#
+# Unless you have an agreement with Numenta, Inc., for a separate license for
+# this software code, the following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+__doc__ = """
+A tutorial that shows some features of the Temporal Memory.
+
+This program has demonstrates some basic properties of the
```

Comment:
An extra `has`.

Reply:
This is a very nice demonstration of TM. 👍

Reply:
Good spot!